### ISSUE TITLE
Parsing problem

### ISSUE THREAD

**Issue by veggiemonk** [dkhamsing/awesome_bot#15]
This is a super nice idea for a project.

Here are some links that were marked as 404 but in the browser, they work.
It ran against [awesome-docker](https://github.com/veggiemonk/awesome-docker)
Note the closing `)` or the `/` at the end.

```
2. 404 https://twitter.com/sebgoa)
3. 404 https://twitter.com/neependra)
4. 404 https://twitter.com/allingeek)
5. 404 https://twitter.com/kartar)
6. 404 http://www.bee42.com/dockerbook/
7. 404 https://twitter.com/adrianmouat)
8. 404 https://twitter.com/PRossbach)
```

Installed with `gem install awesome_bot` and
```
> ruby -v
ruby 2.2.4p230 (2015-12-16 revision 53155) [x86_64-darwin12.0]
```

Cheers

**Comment by dkhamsing** [comment]
Thanks for the report, will investigate

**Comment by dkhamsing** [comment]
@veggiemonk hey I published a new version 1.3.2, can you try it and report back :smile:

**Comment by veggiemonk** [comment]
Cool. Thanks a lot.
